docs/user_guide/notes.rst

```diff
@@ -8,11 +8,11 @@ ngshare should NOT be run concurrently, or there may be race conditions and data
 
 Database Upgrade
 ----------------
-ngshare checks the database version every time it starts up. If the database version is older than the ngshare version, it performs schema and data migration. Normally, this may only happen after ngshare is updated.
+ngshare checks the database version every time it starts up. If the database version is older than the ngshare version, it performs schema and data migration. 
 
-The check can be disabled using ``--no-upgrade-db`` but do not disable it unless you have a good reason and know the possible consequences. 
+Under normal circumstances, migrations only happen after ngshare is updated and the update involves change in database structure. The ngshare database update log can be found in :doc:`/contributer_guide/alembic`.
```

Comment:
```suggestion
Under normal circumstances, migrations only happen after ngshare is updated and the update involves a change in database structure. The ngshare database update log can be found in :doc:`/contributer_guide/alembic`.
```

Reply:
I think "changing database structure" is better.

Reply:
"involves changing the database structure"

Reply:
Fixed